src/commands/misc/socials.ts

```
@@ -0,0 +1,41 @@
+// Copyright (C) 2024-2025 The Queer Students' Association of Te Herenga Waka Victoria University of Wellington Incorporated, AGPL-3.0 Licence.
+
+import { SlashCommandBuilder, ChatInputCommandInteraction, WebhookClient } from "discord.js";
+import { Command } from "../../../@types";
+import settings from "../../../settings.json";
+import { Command } from "../../../@types";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Command'. Consider to delete it or rename it.

'Command' is defined here:


(lint/suspicious/noRedeclare)